mirage/factories/resource.ts

```diff
@@ -0,0 +1,26 @@
+import { Factory } from 'ember-cli-mirage';
+import faker from 'faker';
+
+import ResourceModel from 'ember-osf-web/models/resource';
+
+export default Factory.extend<ResourceModel>({
+    pid() {
+        return faker.internet.url();
+    },
+    name(){
+        return faker.lorem.sentence();
+    },
+    description() {
+        return faker.lorem.paragraph();
+    },
+    resourceType() {
+        return faker.random.arrayElement(['data','materials']);
```

Comment:
Can we use the values of the enum defined in the resource model?

Reply:
Technically, yeah, though I dislike every method of iterating over an enum.

Reply:
Oh, right, it's an enum, not an object... Nevermind then!

Reply:
Looks like it's fine. I was making it harder than it had to be.